install_config/persistent_storage/pod_security_context.adoc

```diff
@@ -158,116 +410,155 @@ Currently the list of volumes which support ownership management includes:
 * Ceph RBD
 * gitRepo
 
-GlusterFS and NFS do not support ownership management.
+[[user-id]]
+== User IDs
+*Note:* the link:#scc[SCC overview], above, should be read before working with user ids.
 
+User ids can be defined in the container image or in the pod spec. In the pod spec, a single user
+id can be defined global to all containers, or specific to individual containers (or both). A user
+id is supplied as shown in the pod fragement below:
+[[pod-user-id-99]]
+```
+spec:
+  containers: <1>
+  - name: ...
+    securityContext:
+      runAsUser: 99  #nobody
+```
+<1> id 99 is container specific and matches the owner id on the export. If the NFS's export's owner id was 54321 then that number would be used in the pod spec.
+ Specifying `securityContext` outside of the container spec makes the id global to all containers in the pod.
+
+Similar to group ids, user ids may be validated according to policies set in the SCC and/or
+namespace. If the SCC's `runAsUser` strategy is set to _RunAsAny_ then any user id defined in
+the pod spec or in the image is allowed. *Note:* this means a UID of 0 (root) is allowed!
+If, instead, the `runAsUser` strategy is set to _MustRunAsRange_ then a supplied user id will
+be validated against a range of allowed ids. If the pod supplies no user id then the default
+id is the minimum value of the range of allowable user ids.
+
+Getting back to the link:#nfs-example[NFS example], the container needs it's UID set to 99,
+which is shown in the pod fragement above. Assuming the _default_ project and the _restricted_
+SCC, the pod's requested user id of 99 will *not* be allowed, and therefore the pod will fail.
+The pod fails because:
+
+- it requests 99 as its user id,
+- since all available SCCs use _MustRunAsRange_ for their `*runAsUser*` strategy, uid range
+checking is required, 
+- 99 is not included in the SCC or in namespace's user id range, so the pod fails.
+
+To fix this situation:
+
+- the _restricted_ SCC could be modified to include 99 within its min and max user id range
+(*not* recommended),
```

Comment:
thus the bold "not recommended" statement... but it is something that the reader might consider and therefore I mention it and say not to do it... Is a stronger warning needed? Is it better to not recognize that an admin might want to modify an existing scc?